README.md

```diff
@@ -3,24 +3,29 @@
 [![CircleCi](https://circleci.com/gh/mozilla/moztelemetry.svg?style=shield&circle-token=3fff2168f7d8da61b47bd1481c4fcb984ec88ef5)](https://circleci.com/gh/mozilla/moztelemetry)
 
 # moztelemetry
+
 Mozilla's Telemetry API for Scala
 
 ## Using moztelemetry
+
 In SBT:
 ```
 resolvers += "S3 local maven snapshots" at "s3://net-mozaws-data-us-west-2-ops-mavenrepo/snapshots"
 libraryDependencies += "com.mozilla.telemetry" %% "moztelemetry" % "1.0-SNAPSHOT"
 ```
 
 ## Testing
-To run the tests you have to start a mock S3 service first with moto:
 
-```
-pip install moto
-moto_server s3 -p 8001 &
-AWS_ACCESS_KEY_ID=foo AWS_SECRET_ACCESS_KEY=foo sbt test
-```
+To run the tests, build the docker container and run the tests.
+
+Build the container. You only have to do this once or when you update the Dockerfile:
+
+	docker build -t moztelemetry .
```

Comment:
Though this works for monospace code font, it's probably good practice that we use the same one throughout the README - here it's ```